app/shop.py

```diff
@@ -0,0 +1,39 @@
+import datetime
```

Comment:
The import statement should be `import datetime` as per the checklist requirement.